zoo-css-challenge/index.html

```diff
@@ -80,52 +83,72 @@ <h3>Height and Size</h3>
 
 		<section class="news">
 			<h2>News</h2>
-			<div class="card">
+			<div class="container">
+			<div class="box">
+				
 				<time>2010-04-16</time>
-				<h4>Helping wild turtles in Seychelles</h4>
+				<h3>Helping wild turtles in Seychelles</h3>
 				<a>Find out More</a>
+				<br>
```

Comment:
```suggestion
```
It's not common to use `<br>` anymore in frontend development. Instead, we would add space using CSS margins